.github/dependabot.yaml

```diff
@@ -0,0 +1,24 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    versioning-strategy: increase
+    schedule:
+      interval: 'daily'
+      time: '09:00'
+      timezone: 'Europe/Berlin'
+    commit-message:
+      prefix: 'deps'
+      prefix-development: 'deps(dev)'
+    groups:
+      sentry:
+        patterns:
+          - '@sentry/*'
```

Comment:
```suggestion
```

I'm not exactly sure what does this part of do but since we're not using sentry let's delete it.

Reply:
I think it instructs Dependabot to update all `@sentry/` dependencies together, in one pull request.

+1 to remove this, I don't expect us to need Sentry in this repo, since it's a library (not an app).